AutoCollection/Performance.ts

```diff
@@ -135,10 +137,26 @@ class AutoCollectPerformance {
                 totalIrq += irq;
             }
 
+            // Calculate % of total cpu time (user + system) this App Process used (Only supported by node v6.1.0+)
```

Comment:
Should we if guard the section above when the section below would be used? Just to avoid pointless arithmetic that we're going to throw away